Modulefile

```diff
@@ -7,4 +7,4 @@ summary 'autofs'
 description 'Manage automounter'
 project_page 'https://github.com/ghoneycutt/puppet-module-autofs'
 
-dependency 'ghoneycutt/nfs', '>= 0.0.1'
```

Comment:
Why no longer the dependency on NFS? I'm ok with that, though this is a breaking change, which is also OK. Just want to understand the why.